asv_bench/benchmarks/series_methods.py

```diff
@@ -151,6 +163,18 @@ def time_mode(self, N, dtype):
         self.s.mode()
 
 
+class ModeObjectDropNAFalse:
+
+    params = [10 ** 3, 10 ** 4, 10 ** 5]
+    param_names = ["N"]
+
+    def setup(self, N):
+        self.s = Series(np.random.randint(0, N, size=10 * N)).astype("object")
```

Comment:
sasme here can you parameterize over the dtype as well

Reply:
I think dropna=False is probably only interesting for "object", thus I didn't parametrize dtype at all.